### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by christinece** [ponlawat-w/moodle-local_accessibility#16]
Hello, 

I've been carrying out some accessibility testing on our Moodle site, which uses the accessibility plugin. With the Wave accessibility testing tool, the test returned three errors (missing form labels) for the accessibility plugin. 

Apologies if you're already aware of this, I thought I should let you know in case you aren't.
